go.mod

```diff
@@ -208,6 +210,7 @@ require (
 	github.com/gordonklaus/ineffassign v0.0.0-20230107090616-13ace0543b28 // indirect
 	github.com/goreleaser/chglog v0.5.0 // indirect
 	github.com/goreleaser/fileglob v1.3.0 // indirect
+	github.com/gorilla/csrf v1.7.1
```

Comment:
does this need `go mod tidy` ?  Typically, direct package dependencies are groups separately from the indirect packages.

Reply:
Running `go mod tidy` didn't seem to change anything - It actually looks like there are no groupings in our current mod file?